kinesis_video_streamer/launch/kinesis_video_streamer.launch.py

```diff
@@ -59,14 +63,49 @@ def generate_launch_description():
     launch.actions.DeclareLaunchArgument(
       CONFIG,
       default_value=default_config
+    ),
+    launch.actions.DeclareLaunchArgument(
+      AWS_REGION,
+      default_value=default_aws_region
+    ),
+    launch.actions.DeclareLaunchArgument(
+      STREAM_NAME,
+      default_value=default_stream_name
+    ),
+    launch.actions.DeclareLaunchArgument(
+      REKOGNITION_DATA_STREAM,
+      default_value=default_rekognition
     )
-   ])
+  ])
+
+  node_parameters = [launch.substitutions.LaunchConfiguration(CONFIG)]
+  node_parameters.append({
```

Comment:
You could create all these at the same time instead of calling append over and over. 

```
node_parameters = [
  launch.substitutions.LaunchConfiguration(CONFIG),
  'aws_client_configuration': {
      'region': launch.substitutions.LaunchConfiguration(AWS_REGION)
    },
    'kinesis_video': {
      'stream0': {
        'stream_name': launch.substitutions.LaunchConfiguration(STREAM_NAME)
      }
    }
```

etc

Reply:
Done.